src/main.ts

```diff
@@ -35,6 +41,9 @@ async function run(): Promise<void> {
     } else {
       process.exit(0);
     }
+  } finally {
+    core.setOutput('jira-issue-found', jiraIssueFound.toString());
+    core.setOutput('jira-issue-source', jiraIssueSource);
```

Comment:
I believe this change is now causing a deprecation warning:

> Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
you are right, I'd add a fix, thank you